### ISSUE TITLE
Create a TreeView Diff View

### ISSUE THREAD

**Issue by mmattner** [mmattner/vPinBackupManager#28]
Currently the treeview shows all files for either the system or backup, but what happens if they differ.

Users CAN flick between backup and live views and expand tree and flick back and forth.

I propose two ways of providing more information:
1. In the existing treeview, if a file exists in one view (Current) or backup, but not the other, show it in green, or with a green background and in the other view that it doesnt exist show it as a blank space, or show it in a light grey font or similar to show it isnt there.
2. If files differ between views show the files in red .... taking a leaf out of the standard diff views for text files, potentially treasting the backup as the "old version" in git terms and the current version as the HEAD.

If this is not enough it could be possible to add a show diffs button on the treeview which launched a modal dialog with the current and backup trees side by side - with a similar colouring style to let people look at deltas.

Once this is done, thought should be given to context sensitive commands to copy files from backup to current and vice versa - this would allow backups to be put in sync file by file if needed.

**Comment by linojon** [comment]
related to this (but maybe a separate issue) i'd like an "expand all" on the tree (maybe via right-click context menu or double-click the "-" ?). And remember the current expansion setting when clicking another table. 

One more thought, ability to switch between tree view and a table view which could get pretty interesting, like columns for live file, another for backup in same view and other data like timestamps, sizes, etc. Ablity to show/hide columns, sort etc, save current preferences in the config file.
